### PR TITLE
Update README.md

### LINKED ISSUE #944: VS Version in the Readme is Outdated
Replace `Visual Studio 2017 ` with `Visual Studio 2019`.

### LINKED ISSUE #957: Outdated README
Resolve #956 for this issue.

### PR DESCRIPTION
**Pull request by lzardy**
Provides more in-depth instructions (because I got bored helping lesser abled people).

Resolve #957
Resolve #944

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,29 +10,34 @@ Project chat on Discord: https://discord.gg/Bz3znAM
...
-(https://github.com/LeagueSandbox/LeagueSandboxAutoSetup/archive/v1.1.tar.gz) [[Mirror]]
+[[Source]](https://github.com/LeagueSandbox/LeagueSandboxAutoSetup/archive/v1.1.zip)
+[[Mirror]](https://github.com/LeagueSandbox/LeagueSandboxAutoSetup/archive/v1.1.tar.gz)
 * Build and run
+* If having trouble, follow the Manual Setup below
 		
 ### Manual Setup (Windows/Mac/Linux)
-* Download the 4.20 version of League client (https://mega.nz/#!hpkiQK5A!pFkZJtxCMQktJf4umplAdPC_Fukt0xgMfO7g3bGp1Io)
-* Clone the git repository and run ```git submodule update --init --recursive``` to download the necessary contents package 
-* Copy `GameServer/Settings/GameInfo.json.template` to `GameServer/Settings/GameInfo.json`
```
Comment (danil179) [inline README.md:27]: Not needed anymore?

File: README.md [hunk 2]
```diff
@@ -10,29 +10,34 @@ Project chat on Discord: https://discord.gg/Bz3znAM
...
-
-#### Automatically Launching from Visual Studio
+* Download the 4.20 version of League game client:
+	1. Unscrubbed, packed version: (https://mega.nz/#!hpkiQK5A!pFkZJtxCMQktJf4umplAdPC_Fukt0xgMfO7g3bGp1Io)
+	2. Scrubbed, moddable version: (https://drive.google.com/open?id=1vr6kGpDK1Hq3Loh8-2z7dlmXSCGKqY2Z)
+* For running the git commands below, [Git Bash](https://gitforwindows.org/) is recommended
+* Clone the git repository using ```git clone https://github.com/LeagueSandbox/GameServer.git```, then to download the necessary contents packages, run:
+	* ```cd GameServer```
+	* ```git submodule init```
+	* ```git submodule update```
```
Comment (danil179) [inline README.md:33]: Is it equivalent to `git submodule --init --recursive`?

Comment (lzardy) [reply README.md:33]: That command does not exist.

File: README.md (hunk 1 above)
Comment (lzardy) [reply README.md:27]: Copying of the template is a useless action as running the GameServerConsole for the first time will auto-create them in the appropriate Settings folder.

**Review by danil179 — APPROVED** [review]

**Comment by danil179** [comment]
Open an issue and add "Resolve ..." to the PR message

**Comment by lzardy** [comment]
Resolve #957.

**Pull request merged**